style.css

```diff
@@ -690,18 +818,22 @@ nav ul li img {
     margin-bottom: 0;
   }
 
+  .grid-container {
+    grid-template-columns: auto;
+  }
+
   .card {
-    display: flex;
     flex-direction: column;
     padding: 16px 16px 24px 16px;
-    height: min-content;
+
+    /* height: min-content; */
```

Comment:
Please ensure that any unused commented codes are removed from the file. Thank you.